ludwig/features/binary_feature.py

```diff
@@ -69,13 +69,13 @@ def forward(self, v: TorchscriptPreprocessingInput):
             v = torch.stack(v)
 
         if torch.jit.isinstance(v, torch.Tensor):
-            return v.to(dtype=torch.bool)
+            return v.to(dtype=torch.float32)
```

Comment:
Curious why do these need to be explicitly casted to float32?

Reply:
It seems that if the input features are comprised exclusively of binary features, torchscript fails.

My best guess: torchscript does not know to upcast these booleans to float32 during the forward pass, therefore causing an error.